website/docs/software/plugins/serial.md

```diff
@@ -0,0 +1,46 @@
+---
+id: serial-plugin
+title: Serial communication plugin
+sidebar_label: Serial communication
+---
+
+## About
+
+A simple interface to send messages over the mesh network by sending strings
+over a serial port.
+
+Default is to use RX GPIO 16 and TX GPIO 17.
+
+
+## Basic Usage:
+
+        1) Enable the plugin by setting serialplugin_enabled to 1.
+        2) Set the pins (serialplugin_rxd / serialplugin_rxd) for your preferred RX and TX GPIO pins.
+           On tbeam, recommend to use:
+                RXD 35
+                TXD 15
+        3) Set serialplugin_timeout to the amount of time to wait before we consider
+           your packet as "done".
+        4) (Optional) In SerialPlugin.h set the port to PortNum_TEXT_MESSAGE_APP if you want to
+           send messages to/from the general text message channel.
+        5) Connect to your device over the serial interface at 38400 8N1.
```

Comment:
I find it easier to read when numbers like 38400 8N1 are in tick marks like this `38400 8N1`

also the variables above (line 24) could potentially be put in marks like that? We need to figure out a style guide.

Reply:
Done

Reply:
Agreed